apps/disease_control/models.py

```diff
@@ -179,7 +180,9 @@ class EmergentHealthTopicListPage(Page):
 
     def get_context(self, request):
         context = super().get_context(request)
-        context["ordered_diseases"] = DiseaseAndConditionDetailPage.objects.filter(
-            is_emergent=True
-        ).order_by("title")
+        context["ordered_diseases"] = (
```

Comment:
Could we also add a test that the objects in `context["ordered_diseases"]` are ordered by their `latest_revision_created_at` field?

Reply:
Pushed a test.